packages/babel-preset-env/src/polyfills/corejs3/entry-plugin.js

```diff
@@ -21,6 +21,9 @@ function isBabelPolyfillSource(source) {
 }
 
 function isCoreJSSource(source) {
+  if (typeof source === "string") {
+    source = source.replace(/(\/(index)?)?(\.js)?$/i, "").toLowerCase();
```

Comment:
I don't think we should use `i` and `toLoweCase`, otherwise imports that work when transpiled stop working if they are left as-is.

Reply:
What do you mean?
![image](https://user-images.githubusercontent.com/2213682/54864637-809c5500-4d8c-11e9-8977-33c082e81d73.png)

Reply:
Not on every OS:

![Schermata da 2019-03-23 11-28-01](https://user-images.githubusercontent.com/7000710/54865092-5c2a8380-4d60-11e9-8625-132c4687afd7.jpg)

Reply:
Mh maybe it is ok to normalize paths; it won't cause any bug.

Reply:
I don't care, it's not something principal for me.